src/Microsoft.PowerShell.ConsoleHost/Microsoft.PowerShell.ConsoleHost.csproj

```diff
@@ -16,6 +16,14 @@
     <DefineConstants>$(DefineConstants);CORECLR</DefineConstants>
   </PropertyGroup>
 
+  <ItemGroup Condition=" '$(Configuration)' == 'Linux' ">
```

Comment:
Configuration `Linux` has been removed.  See [Common Props](https://github.com/PowerShell/PowerShell/blob/948532ac839f3bc7e8bbbd3c9f5f292868ac8687/PowerShell.Common.props#L138) for how to conditionally remove for Non-Windows.

Reply:
Thanks, I addressed that.